src/assets/icons/CloseIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
I need to check in next example if the close icon is displayed. Not sure if I need to do something else regarding the configuration here and there.

Reply:
Should be okay, since it's a react component and not a raw SVG I don't think there's more to do here